onnxruntime/core/framework/session_state_flatbuffers_utils.cc

```diff
@@ -74,4 +77,32 @@ Status FbsSessionStateViewer::GetSubgraphSessionState(NodeIndex node_idx, const
   return Status::OK();
 }
 
+#define SEP ":"
+#define OPINFO(d, o, s) #d SEP #o SEP #s
+
+void UpdateHashForBackwardsCompatibility(HashValue& hash) {
```

Comment:
could we make this code have required reviewers, similar to the expected kernel def hash files? perhaps it could go in its own file.
https://github.com/microsoft/onnxruntime/blob/bfb20b315d9763f4113fdb1b15444626d66874e6/CODEOWNERS#L14
also it might be good to add some info about when one might want to update this map and how to do so.

Reply:
added info.

should we have a file for these and the transpose optimizer hashes? can create and merge things into that once ashwini's PR is checked in